src/Languages/Editor/Test/Idle/IdleTaskQueueTest.cs

```diff
@@ -72,7 +73,7 @@ public class IdleTaskQueueTest {
 
         private void RunThreads() {
             for (int i = 0; i < 10; i++) {
-                _editorShell.DoIdle();
+                ((IIdleTimeSource)_shell).DoIdle();
```

Comment:
Not yet. But we'll change it. Since it is used in many places as `shell.Idle` postponed this to next round. Editor uses it extensively so it will be good time to untangle that.